packages/core/theme-module/theme/components/CartSidebar.vue

```diff
@@ -13,20 +13,20 @@
             <transition-group name="fade" tag="div">
               <SfCollectedProduct
                 v-for="product in products"
-                :key="getCartProductName(product)"
-                :image="getCartProductImage(product)"
-                :title="getCartProductName(product)"
-                :regular-price="getCartProductPrice(product)"
+                :key="cartGetters.getProductName(product).value"
```

Comment:
i think it will look cleaner if you will return it like this:
```
return {
  productName: cartGetters.getProductName(product)
}
```
- you will see imemdiately what is available in a template
- no need to use ugly `value`